error: fedora22 is not supported

Unable to build the client in Fedora 22, running kernel version `4.1.4-200.fc22.x86_64`. I've installed all the dependencies.

Here is the command log.

```bash
$ npm install aerospike --save
npm http GET https://registry.npmjs.org/aerospike
npm http 304 https://registry.npmjs.org/aerospike
npm http GET https://registry.npmjs.org/nan
npm http 304 https://registry.npmjs.org/nan

> aerospike@1.0.49 node_modules/aerospike
> node-gyp rebuild

make: Entering directory 'node_modules/aerospike/build'
  ACTION binding_gyp_aerospike_client_c_target_run_scripts_aerospike_client_c_sh binding_gyp_aerospike_client_c_target_run_scripts_aerospike_client_c_sh.intermediate
error: fedora22 is not supported.
aerospike-client-c.target.mk:15: recipe for target 'binding_gyp_aerospike_client_c_target_run_scripts_aerospike_client_c_sh.intermediate' failed
make: *** [binding_gyp_aerospike_client_c_target_run_scripts_aerospike_client_c_sh.intermediate] Error 1
make: Leaving directory 'node_modules/aerospike/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/lib/node_modules/node-gyp/lib/build.js:267:23)
gyp ERR! stack     at ChildProcess.emit (events.js:98:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:820:12)
gyp ERR! System Linux 4.1.4-200.fc22.x86_64
gyp ERR! command "node" "/usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd node_modules/aerospike
gyp ERR! node -v v0.10.36
gyp ERR! node-gyp -v v0.10.6
gyp ERR! not ok 
npm ERR! weird error 1
npm ERR! not ok code 0
```

--- a/scripts/aerospike-client-c.sh
+++ b/scripts/aerospike-client-c.sh
@@ -51,7 +51,7 @@ detect_linux()
 
     case ${DIST_NAME} in
 
-      "centos6" | "redhatenterpriseserver6" | "centos7")
+      "centos6" | "redhatenterpriseserver6" | "centos7" | "fedora20" | "fedora21" | "fedora22")
         echo "el6" "rpm"
         return 0
         ;;
@@ -79,7 +79,7 @@ detect_linux()
 
     esac
   fi
-  
+
   # no LSB, check for /etc/centos-release
   if [ -f /etc/centos-release ]; then
     dist=$(cat /etc/redhat-release | tr '[:upper:]' '[:lower:]')
@@ -89,7 +89,7 @@ detect_linux()
 	echo "el6" "rpm"
 	return 0
 	;;
-	
+
        * )
 	echo "error: ${dist} is not supported."
 	return 1
@@ -97,7 +97,7 @@ detect_linux()
      esac
    fi
 
-	
+
   # Ok, no LSB, so check for /etc/issue
   if [ -f /etc/issue ]; then
     dist=$(cat /etc/issue | tr '[:upper:]' '[:lower:]')
@@ -173,7 +173,7 @@ download()
   wget_path=`which wget`
   has_wget=$?
 
-  # Check 
+  # Check
   if [ $has_curl != 0 ] && [ $has_wget != 0 ]; then
     echo "error: Not able to find 'curl' or `wget`. Either is required to download the package."
     exit 1
@@ -181,7 +181,7 @@ download()
 
   # Compose the URL for the client tgz
   URL="http://www.aerospike.com/download/client/c/${AEROSPIKE_C_VERSION}/artifact/${PKG_DIST}"
-  
+
   # Download and extract the client tgz.
   # Use non-slient mode to show progress about the download. Important for slower networks.
   printf "info: downloading '${URL}' to '${AEROSPIKE}/package/aerospike-client-c.tgz'\n"
@@ -305,7 +305,7 @@ if [ $DOWNLOAD ] && [ $DOWNLOAD == 1 ]; then
       printf "warning: \n"
     else
       download
-      
+
     fi
 
     ##############################################################################

